### PR TITLE
Fixed issue 498

### LINKED ISSUE #498: Throwing Error when generate a fixed length internet password
**Describe the bug**
Trying to generate a fixed length password include uppercase, special characters and digits.

**To Reproduce**
```
Faker.instance().internet().password(32, 32, true, true, true)
```

**Expected behavior**
Return a password correctly

**Result**

```
java.lang.IllegalArgumentException: bound must be positive

	at java.base/java.util.Random.nextInt(Random.java:388)
	at com.github.javafaker.service.RandomService.nextInt(RandomService.java:24)
	at com.github.javafaker.Lorem.characters(Lorem.java:43)
	at com.github.javafaker.Internet.password(Internet.java:131)
```

**Versions:**
 - OS: Ubuntu 20.04 LTS
 - JDK 11.0.6
 - Faker Version 1.0.2

### PR DESCRIPTION
**Pull request by JinquanPeng**
Fixes #498

### CODE REVIEW

File: src/main/java/com/github/javafaker/Lorem.java [hunk 1]
```diff
@@ -32,15 +32,15 @@ public String characters(boolean includeUppercase) {
     }
 
     public String characters(int minimumLength, int maximumLength) {
-        return characters(faker.random().nextInt(maximumLength - minimumLength) + minimumLength, false);
+        return characters(faker.random().nextInt(maximumLength - minimumLength + 1) + minimumLength, false);
```
Comment (mattmook) [inline src/main/java/com/github/javafaker/Lorem.java:35]: worth having a test that prove these work now?

**Comment by JinquanPeng** [comment]
[LoremTest.zip](https://github.com/DiUS/java-faker/files/4707011/LoremTest.zip)
The test for modified file. Add tests on oringinal file.

File: src/main/java/com/github/javafaker/Lorem.java (hunk 1 above)
Comment (JinquanPeng) [reply src/main/java/com/github/javafaker/Lorem.java:35]: Yeah...this just need to test when `maximumLength = minimumLength` the length of password equals to 'maximunLength'. I attach my test file below.

**Comment by codingricky** [comment]
Should add the test to the PR

**Pull request closed without merging**